src/test/regress/expected/citus_local_tables.out

```diff
@@ -535,6 +535,19 @@ ORDER BY 1;
  unique_a_b_1504027 | citus_local_table_1_1504027 | 1 2
 (2 rows)
 
+-- test creating citus local table with an index from non-default schema
+CREATE SCHEMA "test_\'index_schema";
+CREATE TABLE "test_\'index_schema".testindex (a int, b int);
+CREATE INDEX ind ON "test_\'index_schema".testindex (a);
+ALTER TABLE "test_\'index_schema".testindex ADD CONSTRAINT fkey_to_dummy_ref FOREIGN KEY (a) REFERENCES dummy_reference_table(a);
+NOTICE:  executing the command locally: SELECT worker_apply_inter_shard_ddl_command (1504040, E'test_\\''index_schema', 1504037, 'citus_local_tables_test_schema', E'ALTER TABLE "test_\\''index_schema".testindex ADD CONSTRAINT fkey_to_dummy_ref FOREIGN KEY (a) REFERENCES dummy_reference_table(a);')
+SELECT indexname FROM pg_indexes WHERE tablename LIKE 'testindex%' ORDER BY indexname;
+  indexname
+---------------------------------------------------------------------
+ ind
+ ind_1504040
```

Comment:
seems that shardId's here are changing a lot when mergeing community to enterprise (see https://github.com/citusdata/citus-enterprise/pull/538/commits/a9432ed8de7b9f5ac3d4d751d07a28cbc3de689d)

can you do something like that:
`SELECT COUNT(*)=2 FROM pg_indexes WHERE tablename LIKE 'testindex%' AND indexname LIKE 'ind%';`